scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/yaml/YAMLJavaFieldInfoCreator.java

```diff
@@ -0,0 +1,51 @@
+
+package org.scijava.ops.engine.yaml;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Map;
+
+import org.scijava.ops.api.OpInfo;
+import org.scijava.ops.engine.matcher.impl.OpFieldInfo;
+import org.scijava.ops.engine.matcher.impl.OpMethodInfo;
+
+public class YAMLJavaFieldInfoCreator implements YAMLOpInfoCreator {
+
+	@Override
+	public boolean canCreateFrom(String source, String identifier) {
+		return source.equals("Java") && identifier.indexOf('$') != -1;
```

Comment:
Using `$` as the sole indicator of a field is problematic since it also includes inner classes. You could use the case convention to distinguish the two but that's not foolproof.

I added [an inner class test](https://github.com/scijava/incubator/commit/ab48b870892238b002369ccb3a004dd2c7dd882d) to get passing.

Reply:
@hinerm we *could* just another character (e.g. `_`), the `source`s  syntax that we use in the YAML is really up to us.

Reply:
>we could just another character (e.g. _), the sources syntax that we use in the YAML is really up to us.

@gselzer it's true but I would like to avoid magic characters unless we have no other choice.